### ISSUE TITLE
Discourse - Hamburger menu results per category are not being communicated to screenreader users

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#123]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: Het aantal resultaten voor de categorieën in het hamburgermenu in de hoofdnavigatie wordt niet hardop voorgelezen aan de screenreadergebruiker wanneer ze gefocust zijn op de bijbehorende link. Belangrijke informatie moet programmatisch gekoppeld zijn aan gerelateerde inhoud.

Mogelijke verbeteringen: Voeg een aria-label toe aan de container-link die zowel de naam van de link als het aantal resultaten bevat. Bijvoorbeeld: `<a class="badge-wrapper bullet" href="/c/artikelen/24" aria-label="Artikelen, 9 resultaten">`. Voeg tegelijkertijd een aria-hidden attribuut toe aan het aantal resultaten zodat dit getal genegeerd wordt door screenreaders (die deze informatie al hebben gekregen via het aria-label hierboven). Bijvoorbeeld: `<b class="topics-count" aria-hidden="true">9</b>`

**Referentiecode:**
`<li class="category-link category-artikelen"><span><a class="badge-wrapper bullet" href="/c/artikelen/24"><span class="badge-category-bg" style="background-color: #1042DD;"></span><span data-drop-close="true" class="badge-category clear-badge" title="Blogposts, nieuwsitems en andere artikelen."><span class="category-name">Artikelen</span></span></a></span><b class="topics-count">9</b></li>`

**Gerelateerde WCAG:**
WCAG 1.3.1 Info and relationships
Level A

**Referentieafbeelding:**

![The Focused on Artikelen category does not have the number of results for it communicated to screenreader users](https://github.com/user-attachments/assets/0344c458-99c8-453d-8981-1ee173a745a4)

_**Referentienummer:** A126_
